cst438-project-airline/src/main/java/cst438/domain/SeatRepository.java

```diff
@@ -25,5 +25,10 @@
   @Query(value = "UPDATE seat s SET s.available = 0 WHERE s.seat_id = :id", nativeQuery = true)
   void setSeatToUnavailable(@Param(value = "id") int id);
 
+  @Transactional
+  @Modifying
+  @Query(value = "UPDATE seat s SET s.available = 1 WHERE s.seat_id = :id", nativeQuery = true)
```

Comment:
What is `:id`

Reply:
It is the equivalent of "?1" for update or delete queries. It is specifying the name of the parameter.